src/components/PatchActionButtons/DisablePatch.tsx

```diff
@@ -0,0 +1,41 @@
+import { useMutation } from "@apollo/client";
+import { DropdownItem } from "components/ButtonDropdown";
+import { useToastContext } from "context/toast";
+import {
+  SetPatchPriorityMutation,
+  SetPatchPriorityMutationVariables,
+} from "gql/generated/types";
+import { SET_PATCH_PRIORITY } from "gql/mutations";
+
+interface Props {
+  patchId: string;
+  refetchQueries: string[];
```

Comment:
It looks like this is always "Patch", what's the benefit of passing it in?

Reply:
There probably isn't i just moved this component into its own file and preserved the props.